### ISSUE TITLE
Can slidesdown use reveal.js's plugin set? 

### ISSUE THREAD

**Issue by jaywhj** [slidesdown/slidesdown#13]
hi，can slidesdown use reveal.js's plugin set? 
how to configure it, and is there have a step-by-step demo?

**Comment by jceb** [comment]
Hi @jaywhj, slidesdown has tons of reveal.js plugins integrated. However, you can't add plugins from the markdown file because of security implications for the online version. The online version at https://slidesdown.github.io/ allows you to reference any markdown on the internet and load it as a presentation.

Having said that, do you have any plugins you're missing? We could easily add them to the slidesdown distribution and add switches inside markdown to turn them on / off.

**Comment by jaywhj** [comment]
> Hi @jaywhj, slidesdown has tons of reveal.js plugins integrated. However, you can't add plugins from the markdown file because of security implications for the online version. The online version at https://slidesdown.github.io/ allows you to reference any markdown on the internet and load it as a presentation.
> 
> Having said that, do you have any plugins you're missing? We could easily add them to the slidesdown distribution and add switches inside markdown to turn them on / off.

Yes, I saw it, slidesdown has tons of reveal.js plugins integrated. 
But there are also many useful open source plugins, such as [ reveal.js-menu](https://github.com/denehyg/reveal.js-menu), [reveal.js-toolbar](https://github.com/denehyg/reveal.js-toolbar)...  and sometimes  i also want to customize some CSS styles or preprocessing scripts.
If it's an offline version, can I use the extension feature?

**Comment by jceb** [comment]
CSS you can customize by either adding a style tag in markdown or by adding tailwind/unocss classes to markdown/HTML elements.
We can add useful plugins that you can turn on/off in the markdown frontmatter. Hope this helps. Custom JS loading is off limits.

**Comment by jaywhj** [comment]
> CSS you can customize by either adding a style tag in markdown or by adding tailwind/unocss classes to markdown/HTML elements.
> We can add useful plugins that you can turn on/off in the markdown frontmatter. Hope this helps. Custom JS loading is off limits.

Okay, I'll try it.
thanks.
